libr/io/p/io_ihex.c

```diff
@@ -423,6 +423,7 @@ static bool __resize(RIO *io, RIODesc *fd, ut64 size) {
 RIOPlugin r_io_plugin_ihex = {
 	.name = "ihex",
         .desc = "Intel HEX file (ihex://eeproms.hex)",
+		.uris = "ihex://file.hex",
```

Comment:
broken indent

Reply:
I will fix the indentation on all the files.